File: CSIT111/Lecture_2/Lecture_Labs/Lecturer_Examples/TestSwap_2.py
class TestSwap_2:
    def main(self):
        
        m = 123
        n = 456
        
        print("Before the swaps")
        print("\nm = ", m)
        print("\nn = ", n)
        
        # How to swap? We need to use some temporary location(s)
        # to store the information
        temp = m
        m = n
        n = temp
        
        print("\nAfter the swaps")
        print("\nm = ", m)
        print("\nn = ", n)
        
testswap_2 = TestSwap_2()
testswap_2.main()

File: CSIT111/Lecture_2/Lecture_Labs/Lecturer_Examples/TestSwap_2.java
// How to swap two values in Java?
// 
// File name: TestSwap_2.java
class TestSwap_2
{
	public static void main (String [ ] args)
	{
		int m = 123;
		int n = 456;
		
		System.out.println ("Before the swaps");
		System.out.println ("\tm = " + m);
		System.out.println ("\tn = " + n);
		
		// How to swap? We need to use some temporary location(s)
		// to store the information
		int temp = m;
		m = n;
		n = temp;;
		
		System.out.println ("\nAfter the swaps");
		System.out.println ("\tm = " + m);
		System.out.println ("\tn = " + n);
		
	}
}